packager/packager.cc

```diff
@@ -2089,39 +2088,28 @@ void packageRunCore(core::GlobalState &gs, WorkerPool &workers, absl::Span<ast::
     {
         Timer timeit(gs.tracer(), "packager.rewritePackagesAndFiles");
 
-        auto taskq = std::make_shared<ConcurrentBoundedQueue<size_t>>(files.size());
-
-        for (size_t i = 0; i < files.size(); ++i) {
-            taskq->push(i, 1);
-        }
-
         if constexpr (buildPackageDB) {
             if (gs.packageDB().enforceLayering()) {
                 ComputePackageSCCs::run(gs);
             }
         }
 
-        workers.multiplexJobWait("rewritePackagesAndFiles", [&gs = as_const(gs), &files, taskq]() {
-            Timer timeit(gs.tracer(), "packager.rewritePackagesAndFilesWorker");
```

Comment:
I am sad that we lose the ability to add timings for individual workers, but maybe it's not really valuable to know how long the workers take, and what you really care about would be accomplished with:

```c++
{
  Timer timeit(gs.tracer, "work");
  workers.multiplexJobWait(...);
}
```

instead?

Reply:
I think I agree with that. We can always put timers into the body of the worker to know how long the individual iterations take, and use the span around the whole job as a reference.

Reply:
Wait so do we lose anything?

Could we not have kept the behavior by putting the `Workers` timer in the body?

I'm trying to understand the change in what's not possible with this.

Reply:
Putting it in the body means we'll see many small spans on the thread when viewing the trace, instead of one long span that covers the duration of the work. I think this is an okay tradeoff, as we can wrap one big span around the use of `iterate`, and then see spans for work items on the threads.

Reply:
oh, yeah I don't think I care too much about that.